tests/manager/manager-executeWriteCommand_error-002.phpt

```diff
@@ -22,44 +22,11 @@ try {
     $manager->executeWriteCommand(DATABASE_NAME, $command, ['writeConcern' => new MongoDB\Driver\WriteConcern("undefined")]);
 } catch (MongoDB\Driver\Exception\CommandException $e) {
     printf("%s(%d): %s\n", get_class($e), $e->getCode(), $e->getMessage());
-    var_dump($e->getResultDocument());
 }
 
 ?>
 ===DONE===
 <?php exit(0); ?>
---EXPECTF--
+--EXPECT--
```

Comment:
In addition to "operationTime" appearing at the top of the reply document on 4.0.0-rc5, the order of "ok" and "writeConcernError" have also been switched. Testing for that shuffle would require a very ugly regex.

This test is only asserting that an error is thrown, so I think we're better off ignoring the result document and simply checking the exception with an `EXPECT`.